gson/src/main/java/com/google/gson/stream/JsonReader.java

```diff
@@ -1221,44 +1294,92 @@ public void close() throws IOException {
   }
 
   /**
-   * Skips the next value recursively. If it is an object or array, all nested
+   * <p>Reads the next value recursively and returns the JSON value as a String.
+   * This method is intended for use when the JSON token stream contains a 
+   * value for which deserialization must be deferred.</p>
+   * 
+   * <p>Prefer {@link #skipValue()} when the value would be discarded as it is
+   * faster and uses less memory.</p>
+   * @return The next value in the JSON input. Not {@code null} but possibly
+   * {@code "null"}. Not {@code ""} but possibly {@code "\"\""}. Any escape 
+   * sequences in {@link com.google.gson.stream.JsonToken.STRING String} values
+   * are left intact but whitespace is removed.
+   * @see #skipValue()
+   * @since 2.8.6
+   */
+  public String nextRawJSONValue() throws IOException {
+	  return nextValue(true);
+  }
+  
+  /**
+   * <p>Skips the next value recursively. If it is an object or array, all nested
    * elements are skipped. This method is intended for use when the JSON token
-   * stream contains unrecognized or unhandled values.
+   * stream contains unrecognized or unhandled values.</p>
+   * 
+   * <p>Faster than {@link #nextValue()}.</p>
+   * @see #nextValue()
```

Comment:
Should these link to `nextRawJSONValue()`?

Reply:
Yes you are correct. I'll fix that, thanks.